CP02PMCI/CP02PMCI_R00011_ingest.csv

```diff
@@ -0,0 +1,11 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/cg_data/stc1/imm/adcpt/adcpt*.DAT,CP02PMCI-RII01-02-ADCPTG010,recovered_host,Expected,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/instruments/ADCP*/*RDI_000.000,CP02PMCI-RII01-02-ADCPTG010,recovered_inst,Expected,

```

Comment:
This is correct, but do we want to use the more generic *.000 rather than *RDI_000.000?

Reply:
Agreed! changed